static/js/components/material/Dialog.js

```diff
@@ -71,11 +83,9 @@ export default class Dialog extends React.Component {
         </section>
         <footer className="mdc-dialog__footer">
           <Button type="button" className="mdc-dialog__footer__button mdc-dialog__footer__button--cancel cancel-button">
-            Cancel
-          </Button>
-          <Button type="button" className="mdc-dialog__footer__button mdc-dialog__footer__button--accept edit-button">
-            Save
+            {cancelText ? cancelText : 'Cancel'}
           </Button>
+          {submitBtn}
```

Comment:
how do you feel about inlinining the ternary here? something like

```js
{ noSubmit
  ? null
  : <button></button> }
```

I just prefer this pattern over binding variables above, which to me is a little less direct.

Reply:
OK